_posts/2016-05-29-IMSI-Catching-Catching.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: page
+title: "The State of IMSI Catching Catching"
+#
+# Content
+#
+author: freddy_martinez
+teaser: "Recently, the Lucy Parsons Labs was sent an email about detecting IMSI catchers, often called Stingrays, at protests. Our experience detecting IMSI catchers in the last year is below."
+categories:
+  - stingray
+tags:
+  - technology
+  - surveillance
+#
+# Styling
+#
+header: no
+image:
+ thumb: "thumbs/StingrayThumb.jpg"
+---
+
+This is an area with which I have a good bit of experience. I have spoken about on the topic at conferences including [DEF CON](https://www.youtube.com/embed/JyTb5mJOYL), [THOTCON](https://github.com/freddymartinez9/securitytalks/tree/master/Thotcon) and wrote a [guide for protests.](https://github.com/freddymartinez9/securitytalks/blob/master/IMSICatchersForActivists.md). The Lucy Parsons Labs has spent over a year seeking evidence for IMSI Catcher use at Chicag-area demonstrations. 
```

Comment:
s/, [THOTCON]/ and [THOTCON]/; s/evidence for/evidence of/; s/Chicag-area/Chicago-area/;